prow/cmd/checkconfig/main_test.go

```diff
@@ -1252,10 +1252,25 @@ func TestValidateInRepoConfig(t *testing.T) {
 			prowYAMLFileName = tempFile.Name()
 		}
 
-		cfg := &config.Config{
-			ProwConfig: config.ProwConfig{PodNamespace: "my-ns"},
+		// Need an empty file to load the config from so we go through its defaulting
+		tempConfig, err := ioutil.TempFile("/tmp", "prow-test")
+		if err != nil {
+			t.Fatalf("failed to get tempfile: %v", err)
+		}
+		defer func() {
+			if err := tempConfig.Close(); err != nil {
```

Comment:
We should close this file before `config.Load()` is called (we can close the temp file immediately after creating it).